udp-native/src/main/java/org/ballerinalang/stdlib/udp/UdpClient.java

```diff
@@ -0,0 +1,70 @@
+package org.ballerinalang.stdlib.udp;
+
+import io.ballerina.runtime.api.Future;
+import io.netty.bootstrap.Bootstrap;
+import io.netty.channel.Channel;
+import io.netty.channel.ChannelFutureListener;
+import io.netty.channel.ChannelInitializer;
+import io.netty.channel.EventLoopGroup;
+import io.netty.channel.nio.NioEventLoopGroup;
+import io.netty.channel.socket.DatagramPacket;
+import io.netty.channel.socket.nio.NioDatagramChannel;
+import io.netty.handler.timeout.IdleStateHandler;
+
+import java.net.InetSocketAddress;
+import java.net.SocketAddress;
+import java.util.concurrent.TimeUnit;
+
+/**
+ *  {@link UdpClient} creates the udp client and handles all the network operations.
+ *
+ */
+public class UdpClient {
+
+    private final EventLoopGroup group;
+    private final Channel channel;
+
+    public UdpClient(InetSocketAddress localAddress) throws  InterruptedException {
+        group = new NioEventLoopGroup();
+        Bootstrap b = new Bootstrap();
```

Comment:
Let's not use single letter variable names. Shall we rename this to clientBootstrap ?

Reply:
refactored